system/settings/settings.xml

```diff
@@ -1008,18 +1008,21 @@
           <default>false</default>
           <control type="toggle" />
         </setting>
-        <setting id="videolibrary.groupsingleitemsets" type="boolean" label="20470" help="36157">
+        <setting id="videolibrary.groupsingleitemsets" type="boolean" parent="videolibrary.groupmoviesets" label="20470" help="36157">
           <level>1</level>
           <default>false</default>
           <control type="toggle" />
+          <dependencies>
+            <dependency type="enable" setting="videolibrary.groupmoviesets" operator="is">true</dependency>
+          </dependencies>
         </setting>
```

Comment:
How it looks

![image](https://cloud.githubusercontent.com/assets/5781142/16179065/ff7478b6-3652-11e6-9ef3-2cbdf18d9b0a.png)